text/2021-04-09-.awsssmchaosrunner integration for AWSChaos.md

```diff
@@ -0,0 +1,24 @@
+# Title
+Integrating awsssmchaosrunner with chaos-mesh, as part of enriching aws-chaos
+## Summary
+
+This will be a kotlin based CLI to run the awsssmchaosrunner, controller of awschaos will create a pod with that “kotlin cli image”, and send commands to inject chaos into EC2 instances
+
+## Motivation
+awsssmchaosrunner, released by asw opensource provides various functionalities to inject chaos in EC2 instance through SSM agent.
+Integrating it with chaos-mesh would help us to add all the failure injections provided by awsssmchaosrunner in chaos-mesh.
+
+## Detailed design
+There would be two parts to it
+1. The kotlin based CLI: awsssmchaosrunner being a kotlin library, we need to wrap it with a user-interface with functionalities of using SSM agent to inject the chaos. This user-interface would then be containerized as a docker image.
+2. The controller part: The controller for the aws chaos like other chaos in chaos-mesh would be written in GOlang and in turn will create a pod with kotlin CLI image to send commands to inject the chaos. 
+
+## Drawbacks
+
```

Comment:
```suggestion
- It will bring one big docker image for kotlin cli (about hundred MiB with JVM)
- After digging in the awsssmchaosrunner project, the core logics are several shell commands that inlined in the kotlin codes, that might be unmaintainable.
```